lib/application.ex

```diff
@@ -165,6 +165,7 @@ defmodule Commanded.Application do
   """
 
   @type t :: module
+  @type options :: [[name: nil | atom] | Supervisor.init_option()]
```

Comment:
Only the `:name` option is currently supported by the [Commanded application supervisor](https://github.com/commanded/commanded/blob/master/lib/commanded/application/supervisor.ex#L39). The supervisor init options (`:strategy`, `:max_restarts`, and `:max_seconds`) aren't supported.

```suggestion
  @type options :: [{:name, atom() | nil}]
```

Reply:
I fixed it, 

I am also a bit confused, https://github.com/commanded/commanded/blob/9b7e89416f1f9a9bff20e458cb445768dbce30c9/lib/commanded/application/supervisor.ex#L39-L45

You pass the entire `init_option` down based on that code, no?

If you don't mind to help me to clarify myself.

Reply:
It looks as though `:name` is the only option supported by the [`Supervisor.start_link/3` function](https://hexdocs.pm/elixir/Supervisor.html#start_link/3) and currently only the [`:strategy` option is passed to the `Supervisor.init/2` function](https://github.com/commanded/commanded/blob/9b7e89416f1f9a9bff20e458cb445768dbce30c9/lib/commanded/application/supervisor.ex#L62). We could extend it to support the additional `:max_restarts` and `:max_seconds` options if required.